README.md

```diff
@@ -1,34 +1,27 @@
-# contained.af
+# Container Escape Bounty CTF version of contained.af
 
-[![Travis CI](https://img.shields.io/travis/genuinetools/contained.af.svg?style=for-the-badge)](https://travis-ci.org/genuinetools/contained.af)
-[![GoDoc](https://img.shields.io/badge/godoc-reference-5272B4.svg?style=for-the-badge)](https://godoc.org/github.com/genuinetools/contained.af)
-[![Github All Releases](https://img.shields.io/github/downloads/genuinetools/contained.af/total.svg?style=for-the-badge)](https://github.com/genuinetools/contained.af/releases)
+This is a fork of [contained.af](https://contained.af/) (a game for learning about containers, capabilities, and syscalls).
```

Comment:
You could add a link to https://github.com/genuinetools/contained.af, in addition to https://contained.af/.

Reply:
Yes, thanks